CHANGELOG.md

```diff
@@ -10,7 +24,7 @@ and macOS), you need either
 
 **Merged pull requests:**
 
-- Remove macOS Monterey requirement from `Package.swift` ([#144](https://github.com/swiftwasm/JavaScriptKit/pull/144)) via [@MaxDesiatov](https://github.com/MaxDesiatov)
```

Comment:
I'd prefer to keep these links so that they work when viewing this Markdown as rendered outside of GitHub. BTW, you can get these changelog snippets generated with links automatically with [GitBuddy](https://github.com/WeTransfer/GitBuddy)

Reply:
They're kept, only moved to bottom as reference links

Reply:
Ah, great. Sorry I missed that.